### ISSUE TITLE
Channel to brainstorm classical concepts that have decent mapping to programming concepts

### ISSUE THREAD

**Issue by MerakDipper** [LingDong-/wenyan-lang#494]
## Describe your issue/suggestions
愚以为wenyan-lang之妙不在其实用性，而在其之信达雅也，若因用而损意，则得不能偿失。与其苛求功能之广，未如先脑风何种古典概念有相映成趣的程式概念？抛砖。“记”可对“log”，“阵”可对“多维数组”，“连环”可对“linked list”云云。

## Checklist

Before you create this issue, Please make sure ALL the following items are checked

- [ ] If this is a feature request, it's NOT listed [Here](https://github.com/LingDong-/wenyan-lang#feature-requests)
- [ ] If this is a bug report, it's NOT listed [Here](https://github.com/LingDong-/wenyan-lang#known-bugs)

**Comment by LingDong-** [comment]
Great idea! see also #349. Discussion are welcome.

**Comment by MerakDipper** [comment]
@LingDong- should there be some doc to collect such ideas? I feel github issue is not good at keeping track of those if more people want to add their mappings. Or let me know if you think otherwise/have better ideas.

**Comment by LingDong-** [comment]
@MerakDipper 
Yes, perhaps we can utilize the wiki page on Github, make a big table in there, and allow everyone to edit? Another convenient way is to use Google Sheets, but unfortunately most people cannot access that.
Do you have a particular service in mind that could be helpful?

**Comment by antfu** [comment]
I would give a vote for GitHub wiki 😃

**Comment by cuixiping** [comment]
How about pin an issue to the top of issue list? Issue list is the easiest place to get attention.

**Comment by LingDong-** [comment]
@cuixiping Didn't know issues can be pinned! That's pretty cool.

However, it seems that non-collaborators cannot edit other people's issue/comment content, so if each people makes a new comment on each translation it would end up a really long thread. One solution is that I look at everybody's comment, and arrange them in a table at the top of the post.

Let me know what you guys prefer

**Comment by cuixiping** [comment]
> @cuixiping I didn't know issues can be pinned! That's pretty cool.
> 
> However, it seems that non-collaborators cannot edit other people's issue/comment content, so if each people makes a new comment on each translation it would end up a really long thread. One solution is that I look at everybody's comment, and arrange them in a table at the top of the post.
> 
> Let me know what you guys prefer

That's a good way. Also you can give the link to wiki page if there is.

**Comment by lymslive** [comment]
Pinned issue is good.
Directly editable page for every is not so good at the beginning of free collection.

**Comment by MerakDipper** [comment]
I was thinking about google sheet before Lingdong pointed that the accessibility issue :) I also wasn't aware of pinned issue - sounds like a good plan to start with. Having the owner refine/remix the pinned notes also help establish the styling and consistency. If it becomes unmanageable for the owner we can then move to a co-edits mode.

**Comment by LingDong-** [comment]
#506 A pinned issue for this is now added. Let's see how it works! Please contribute :D
